pypdf/_text_extraction/_layout_mode/_font.py

```diff
@@ -66,8 +67,9 @@ def __post_init__(self) -> None:
                         # warning and or use reader's "strict" to force an ex???
                         continue
                     # check for format (1): `int [int int int int ...]`
-                    if isinstance(_w[idx + 1], Sequence):
-                        start_idx, width_list = _w[idx : idx + 2]
+                    w_1 = _w[idx + 1].get_object() if isinstance(_w[idx + 1], IndirectObject) else _w[idx + 1]
```

Comment:
Shouldn't

```suggestion
                    w_1 = _w[idx + 1].get_object()
```

be sufficient here?

Reply:
If Sequence.get_object() is safe and just returns the Sequence itself, then yes. I don't have much familiarity with this, but it seems like it does, and you would know better than I would.